src/content/docs/dropins/checkout/slots.mdx

```diff
@@ -12,12 +12,77 @@ Learn about the slots provided in the checkout drop-in component.
 
 [Extending drop-in components](/dropins/all/extending/) describes the default properties available to all slots.
 
+## Title slots
+
+The slots for the [`Title`](/dropins/checkout/containers/overview/#title-and-heading-customization) component are defined in the `TitleProps` interface shown below. The slots allow you to customize the content to be displayed on the title for a specific container.
+
+```ts
+export interface TitleProps {
+  displayTitle?: boolean;
+  slots?: {
+    Title?: SlotProps;
+  };
+}
+```
+
+### Title
+
+The `Title` property is a callback function used to render a custom title content. It receives the default context implementing the `DefaultSlotContext` interface:
+
+```ts
+interface DefaultSlotContext<T> extends PrivateContext<T> {
+  dictionary: Lang;
+  getSlotElement: (key: string) => SlotElement;
+  replaceWith: MutateElement;
+  appendChild: MutateElement;
+  prependChild: MutateElement;
+  appendSibling: MutateElement;
+  prependSibling: MutateElement;
+  onRender: (cb: (next: T & DefaultSlotContext<T>) => void) => void;
+  onChange: (cb: (next: T & DefaultSlotContext<T>) => void) => void;
+}
+
+export type SlotProps<T = any> = (
+  ctx: T & DefaultSlotContext<T>,
+  element: HTMLDivElement | null
+) => Promise<void> | void;
+```
+
+<Aside type="note">
+Some containers are extending their props interface to use the `TitleProps` slot in order to customize the title content. You will find some examples on those specific containers.
+</Aside>
+
+## LoginForm slots
+
+The slots for the [`LoginForm`](/dropins/checkout/containers/login-form/) container are defined in the `LoginFormProps` interface shown below. The slots allow setting the heading content dynamically based on the user authentication status.
```

Comment:
The `LoginFormProps` interface defines the available slots for the [`LoginForm`](/dropins/checkout/containers/login-form/) container. The slots allow setting the heading content dynamically based on the user authentication status.